package/gluon-mesh-vpn-wireguard/src/gluon-hex-to-b64.c

```diff
@@ -0,0 +1,102 @@
+// SPDX-FileCopyrightText: 2022 Jan-Niklas Burfeind <gluon@aiyionpri.me>
+// SPDX-License-Identifier: BSD-2-Clause
+// SPDX-FileContributor: read_hex() by Matthias Schiffer <mschiffer@universe-factory.net>
+
+#include <libubox/utils.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/** return how many samesized elements an array has */
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
+
+/**
+ * how many blocks should be encoded at once - can be configured
+ */
+#define BLOCK_AMOUNT 32
+
+/**
+ * smallest possible block size to encode in b64 without further contex
+ * is three bytes - do not change
+ */
+#define BLOCK_SIZE 3*BLOCK_AMOUNT
```

Comment:
This is the size of 32 blocks, which is kind of counter intuitive. Does it make sense to call it CHUNK_SIZE or so? Then one chunk would include 32 blocks of 3?

Reply:
There may also be another option to clear this up. But how it is now, it's kind of confusing imho.

Reply:
resolved in 89e83277c3fb370e5d31bfe829d1af7a44c4a381